### DISCUSSION TITLE
Inadequate modeling of FIGIs and Tickers

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1406] [Securities (SEC)]
See https://www.openfigi.com/assets/local/figi-allocation-rules.pdf p6 " An identifier is assigned to instruments of all asset classes, is unique to an individual instrument
and once issued will not change for an instrument. For equity instruments an identifier is issued per
instrument per trading venue"
This is in addition to the FIGI for the share issue.
The latest text added by #1245 incorrectly states that this is unusual "		<fibo-fnd-utl-av:explantoryNote>While in most cases, a FIGI uniquely identifies a security, there are situations outside of the U.S. where it instead identifies a listing for a security, similarly to a ticker symbol."

See for example https://www.openfigi.com/search#!?simpleSearchString=zillow&marketSector=Equity&facets=SECURITY_TYP:,,Common%20Stock&page=1&filters=NAME:..ZILLOW%20GROUP%20INC%20-%20A which shows 97 FIGIs for Zillow Class A shares, many of which are in the US (Exchange codes starting U).

More generally each FIGI Exchange or Composite level FIGI needs to reference the Exchange or the Country respectively.

Likewise TickerSymbols are unique (at a given time) to a specific Exchange. There are different syntaxes in use by different publishers (e.g. Fidelity, Yahoo Finance) to qualify the ticker with the exchange.

**Comment by ElisaKendall** [comment]
@rivettp I think you've misunderstood the change made by https://github.com/edmcouncil/fibo/pull/1245.  A listed security is a security that is registered on an exchange.  We have not modeled the complete structure of the FIGI in FIBO but could do so by mapping to the FIGI ontology.  If that is what you would like to see done, then once the next revision of the FIGI standard is published we should consider doing so.   

Having said this, our current representation of a FIGI says that it identifies  either an instrument or a listing.  That instrument may be exchange specific or  country-specific or neither - i.e., a common share that is the most general level.

The definition of TickerSymbol is that it is a reassignable identifier that identifies either a listed security (i.e. one that is exchange-specific) or a listing, which is also exchange specific.  

I don't see an issue here.

**Comment by ElisaKendall** [comment]
We also confirmed with Bloomberg that on occasion, such as in Singapore, the FIGI identifies a listing for the instrument.  It's not the norm, but happens in a few locations such as the Singapore exchange, including the example you provided.  In those cases, the instrument has the same ISIN, and is the same listed security, but with a distinct lot size.  It is the same instrument, however. The work-around that Bloomberg came up with is that the FIGI in those cases distinguishes the listings and their data team acknowledged the anomaly.

**Comment by ElisaKendall** [comment]
So - are you asking us to duplicate the FIGI ontology, or include a mapping for it?  The current solution works with reasoning and the examples we have encoded to date.

**Comment by rivettp** [comment]
I'm not saying we should duplicate FIGI but I'm still confused by the definition of ListedSecurity that says "registered security listed on at least one exchange" which differs from what you just wrote @ElisaKendall "A listed security is a security that is registered on an exchange.". So that at least needs to be clarified.
Maybe if you could send back what should be a very quick update to the TTL file (with FIGIs and tickers) I sent you that would make things clearer.

**Comment by jfgemski** [comment]
You’re both confused.  Registration and listing are two different things.  From Investopedia:

 

A registered security is either a security whose owner is kept on file with the issuer or a security whose transfer is restricted. Registered securities can be the name given to securities whereby ownership is registered with the issuing company or their agent. This is in contrast to bearer securities.

 

Before  <https://www.investopedia.com/terms/s/security.asp> securities—like  <https://www.investopedia.com/terms/s/stock.asp> stocks, bonds, and notes—can be offered for sale to the public, they first must be  <https://www.investopedia.com/terms/r/registration.asp> registered with the  <https://www.investopedia.com/terms/s/sec.asp> Securities and Exchange Commission (SEC). Any stock that does not have an effective registration statement on file with the SEC is considered "unregistered." 

 

So you could have a registered security that is not traded on an exchange.  The above also implies that in the US in order to be traded on an exchange, the security must be registered.

 

Thus the statement “A listed security is a security that is registered on an exchange” should be changed to “A listed security is a security that is listed on an exchange.”

 

I think we have definitions of Registered, Bearer and a few others somewhere else in SEC.

 

John

 

Jfgemski@gmail.com <mailto:Jfgemski@gmail.com>  | M: (516) 353-0004
